lib/internal/streams/utils.js

```diff
@@ -172,20 +172,24 @@ function isReadableFinished(stream, strict) {
 
 function isReadable(stream) {
   if (stream && stream[kIsReadable] != null) return stream[kIsReadable];
-  if (typeof stream?.readable !== 'boolean') return null;
+  if (typeof stream?.readable !== 'boolean') return false;
   if (isDestroyed(stream)) return false;
-  return isReadableNodeStream(stream) &&
+  return Boolean(
+    isReadableNodeStream(stream) &&
     stream.readable &&
-    !isReadableFinished(stream);
+    !isReadableFinished(stream)
+  );
 }
 
 function isWritable(stream) {
   if (stream && stream[kIsWritable] != null) return stream[kIsWritable];
-  if (typeof stream?.writable !== 'boolean') return null;
+  if (typeof stream?.writable !== 'boolean') return false
```

Comment:
Keep the linter happy. 😉 
```suggestion
  if (typeof stream?.writable !== 'boolean') return false;
```